pex/bin/pex.py

```diff
@@ -43,168 +49,136 @@ def log(msg, V=0):
         print(msg, file=sys.stderr)
 
 
-def parse_bool(option, opt_str, _, parser):
-    setattr(parser.values, option.dest, not opt_str.startswith("--no"))
-
-
-def increment_verbosity(option, opt_str, _, parser):
-    verbosity = getattr(parser.values, option.dest, 0)
-    setattr(parser.values, option.dest, verbosity + 1)
-
-
-def process_disable_cache(option, option_str, option_value, parser):
-    setattr(parser.values, option.dest, None)
-
-
-class PyPiSentinel(object):
-    def __str__(self):
-        return "https://pypi.org/simple"
-
-
-_PYPI = PyPiSentinel()
-
+_PYPI = "https://pypi.org/simple"
 
-def process_pypi_option(option, option_str, option_value, parser):
-    if option_str.startswith("--no"):
-        setattr(parser.values, option.dest, [])
-    else:
-        indexes = getattr(parser.values, option.dest, [])
-        if _PYPI not in indexes:
-            indexes.append(_PYPI)
-        setattr(parser.values, option.dest, indexes)
 
+_DEFAULT_MANYLINUX_STANDARD = "manylinux2014"
 
-def process_find_links(option, option_str, option_value, parser):
-    find_links = getattr(parser.values, option.dest, [])
```

Comment:
All this did was remove dups, which seems unnecessary, so we just use normal `action='append'` now.